### PR TITLE
chore(version-info): update release notes

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to #3098 

image

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Feel free to already give feedback 👍

**Comment by barmac** [comment]
Do you have a full changelog update anywhere? It would be helpful to provide feedback on this PR.

**Comment by pinussilvestrus** [comment]
I didn't prepare the changelog yet, unfortunately. We agreed on these items recently (cf. https://camunda.slack.com/archives/GP70M0J6M/p1662020610527819). 

Regarding the modeling improvements, I think you can also use the spring cleaning overview: https://github.com/bpmn-io/internal-docs/issues/607

**Comment by pinussilvestrus** [comment]
Or: https://github.com/camunda/camunda-bpmn-js/blob/main/CHANGELOG.md#0170

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
...
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          With this release, we added generic extension properties supported by Camunda Platform 8.1.
         </li>
         <li>
-          <h4>Diagram errors in the properties panel</h4>
-          All diagram errors are shown in the properties panel as of this release. Open the error panel for the errors to be shown.
+          <h4>Diagram errors on the canvas</h4>
+          Diagram errors are shown on the canvas as of this release. Click on an overlay to display individual element errors in the properties panel.
```
Comment (philippfromme) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: Clicking the overlay doesn't always show an entry in the properties panel (e.g. element type not supported).

How about: _Diagram errors are now shown on the canvas. Click on an error to show details._

**Comment by philippfromme** [comment]
Generally, I don't think we have to include _As of this release_ for every paragraph. It's implicit that it's about this release.

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js]: 👍

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          With this release, we added generic extension properties supported by Camunda Platform 8.1.
```
[the remark this answers is not in the document]
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js]: 👍

**Comment by philippfromme** [comment]
[...] are now sticky _to provide reliable context._ Shouldn't it be _relevant context_ or something similar? What is unreliable context?

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          We now support generic extension properties, which will be available for execution starting from Camunda Platform 8.1.
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:48]: Do users understand what extension properties are? May mention something that makes the link to C7 where users may know them already. But new users may not know what this is—maybe link the documentation

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 4]
```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
...
         </li>
         <li>
-          <h4>Diagram errors in the properties panel</h4>
-          All diagram errors are shown in the properties panel as of this release. Open the error panel for the errors to be shown.
+          <h4>Diagram errors on the canvas</h4>
+          Diagram errors are now shown on the canvas. Click on an error to show details.
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: are now shown **directly** on the canvas

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 5]
```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
...
+        </li>
+        <li>
+          <h4>Sticky properties panel headers</h4>
+          The section headers of the BPMN and DMN properties panels are now sticky to provide reliable context.</li>
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:56]: ...  to provide reliable context when you scroll.

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 6]
```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
...
+        <li>
+          <h4>Modeling UX improvements and bug fixes</h4>
+          This release includes a lot of modeling UX improvements. Sequence flows are now always displayed on top. Also, we fixed several issues to improve the copy and paste behavior.
```
Comment (christian-konrad) [inline client/src/plugins/version-info/ReleaseInfo.js:59]: Do we actually have a lot of improvements?

**Review by christian-konrad — CHANGES_REQUESTED** [review]
See my small annotations

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 6 above)
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js:59]: I'd say so, it includes almost everything from this list: https://github.com/bpmn-io/internal-docs/issues/607

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 3 above)
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js:48]: Do we have documentation for this somewhere? Was not able to find something, yet: https://docs.camunda.io/docs/next/components/modeler/bpmn/tasks/

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 5 above)
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js:56]: @philippfromme _reliable context_ means that users always now reliably in which group they are.

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 3 above)
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js]: What about simply

> You may already be familiar with this feature from Camunda Platform 7.

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 6 above)
Comment (pinussilvestrus) [reply client/src/plugins/version-info/ReleaseInfo.js:59]: Mh, I don't know whether this one is worth a mention. Technically we simply upgraded the Codemirror version, and the search bar now has a bit more functionality. But I don't know whether we want to promote this one at all.

Looking at the current snapshot, we already include a lot of stuff (especially compared to the last release notes).

image


It's mentioned [in the Changelog](https://github.com/camunda/camunda-modeler/pull/3121).

**Comment by pinussilvestrus** [comment]
As suggested by @christian-konrad I removed the C8 extension properties info as you can't select 8.1 as execution platform.

image

**Review by barmac — APPROVED** [review]
LGTM

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 6 above)
Comment (christian-konrad) [reply client/src/plugins/version-info/ReleaseInfo.js:55]: Agree

**Review by christian-konrad — APPROVED** [review]
LGTM

**Pull request merged**